src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { composeStory } from '@storybook/react';
+import { render, screen } from '@testing-library/react';
+import Meta, { Id as IdStory } from './Accordion.stories';
+
+const IdTest = composeStory(IdStory, Meta);
+
+describe('Accorion', () => {
+  test('Add ids', async () => {
+    render(<IdTest />);
+
+    const detailsElement = await screen.findByRole('group');
+    const summaryElement = detailsElement.querySelector('summary');
```

Comment:
I want to use the DisclosureTriangle role to find. But it doesn't seem to work in jsdom.